test/index-test.js

```diff
@@ -13,8 +14,33 @@ export const collect = s => {
 
 describe('hold', () => {
   it('should deliver most recent event to new observer', () => {
-    const s = hold(mergeArray([at(0, 0), at(1, 1), at(2, 2)]))
+    const s = hold(mergeArray([at(0, 0), at(2, 1), at(3, 2)]))
```

Comment:
Notice the slight change here. Using `at(1, 1)` for the second event will cause it to be scheduled fast enough that the "held" value is already `1` and only the events `1 and 2` will be seen.

This might be considered a breaking change.